### PR TITLE
[Position] fix position update

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
- fix position parsing
- properly reassign order groups (and potentially any other future attribute) to bundled orders

### CODE REVIEW

**Review by Herklos — APPROVED** [review]

File: octobot_trading/personal_data/orders/order_util.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def check_cost(total_order_price, min_cost):
     """
     if total_order_price < min_cost:
         if min_cost is None:
-            logging.get_logger().error("Invalid min_cost from exchange")
+            logging.get_logger("order_util").error("Invalid min_cost from exchange")
```
Comment (Herklos) [inline octobot_trading/personal_data/orders/order_util.py:94]: Can we use a common constant for order_util logger name? (duplicated below)

File: octobot_trading/personal_data/orders/orders_manager.py [hunk 2]
```diff
@@ -100,24 +100,54 @@ async def upsert_order_from_raw(self, order_id, raw_order, is_from_exchange) ->
...
+                              f"this should not happen. Order: {pending_order}")
+        self.pending_creation_orders.append(pending_order)
+
+    async def get_and_update_pending_order(self, created_order):
+        pending_order = self._get_pending_order(created_order, True)
+        # TODO refactor to := when cython will support it
+        if pending_order is None:
+            return None
+        await order_util.apply_pending_order_from_created_order(pending_order, created_order)
+        pending_order.is_initialized = False
```
Comment (Herklos) [inline octobot_trading/personal_data/orders/orders_manager.py:125]: I think this should be set in `apply_pending_order_from_created_order`

File: octobot_trading/personal_data/transactions/types/realised_pnl_transaction.py [hunk 3]
```diff
@@ -35,7 +35,7 @@ def __init__(self, exchange_name, creation_time, transaction_type, currency, sym
         self.trigger_source = trigger_source
         self.side = side
         self.transaction_id = f"{self.exchange_name}" \
-                              f"-{str(uuid.uuid4())[:4]}" \
+                              f"-{str(uuid.uuid4())}" \
```
Comment (Herklos) [inline octobot_trading/personal_data/transactions/types/realised_pnl_transaction.py:38]: Why?

**Review by Herklos — COMMENTED** [review]
👍

File: octobot_trading/personal_data/transactions/types/realised_pnl_transaction.py (hunk 3 above)
Comment (GuillaumeDSM) [reply octobot_trading/personal_data/transactions/types/realised_pnl_transaction.py:38]: This is sometimes generating duplicate transaction issues in backtestings when multiple orders are filled at the same time

Comment (GuillaumeDSM) [reply octobot_trading/personal_data/transactions/types/realised_pnl_transaction.py:38]: Is there a reason for keeping only the last 4 digits there or it's just to save space ?

Comment (Herklos) [reply octobot_trading/personal_data/transactions/types/realised_pnl_transaction.py:38]: I don't remember why only the last $

**Review by Herklos — APPROVED** [review]

**Pull request merged**